iamproperty/iamproperty.github.io
# Ds 65/timeline

## Summary of Changes:
1. Timeline vue component
2. CSS for timeline

## Rationale
Timeline component to show linear representation of events.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/timeline

## Ticket(s)
DS-65

## Checklist

## Notes
...
